.github/workflows/deploy-win.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   build_dist:
-    runs-on: windows-2019
+    runs-on: windows-latest
```

Comment:
@partouf what are your thoughts about this change? I understand a little more the ubuntu-* changes below (though I may which to change to 22.04 rather than latest below...better not to have surprises)

Reply:
I'm going to make a new PR that just bumps to ubuntu 22.04. I don't like "latest" as it'll change without warning. I'll leave the windows ones for @partouf